ona_migration_script/migrate_submissions.py

```diff
@@ -0,0 +1,44 @@
+from ona import OnaApiClient
+import settings
+
+FROM_FORM = '15756'  # imaliyethu
+TO_FORM = '23666'  # per report
+
+DEFAULTS = {'msisdn': '', 'fault_status': '', 'logged_date': ''}
+
+client = OnaApiClient(settings.USERNAME, settings.PASSWORD)
```

Comment:
I wonder if it's worth quickly using optparse to make ``username``, ``password``, ``from_form`` and ``to_form`` parameters?